proposals/Exceptions.md

```diff
@@ -10,10 +10,15 @@ on the second proposal, which uses first-class exception types, mainly based on
 the reasoning that it is more expressive and also more extendible to other kinds
 of events.
 
-This proposal requires the [reference types
-proposal](https://github.com/WebAssembly/reference-types/blob/master/proposals/reference-types/Overview.md)
-as a prerequisite, since the [`exnref`](#the-exception-reference-data-type) type
-should be represented as a subtype of `anyref`.
+This proposal requires the following proposals as prerequisites.
+
+- The [reference types proposal](https://github.com/WebAssembly/reference-types/blob/master/proposals/reference-types/Overview.md),
+  since the [`exnref`](#the-exception-reference-data-type) type should be represented as a subtype of `anyref`.
+
+- The [multi value proposal](https://github.com/WebAssembly/multi-value/blob/master/proposals/multi-value/Overview.md),
+  since otherwise the [`br_on_exn?`](#Exception data extraction) instruction would only work with exceptions which consume one value.
+  Moreover, by using [multi value](https://github.com/WebAssembly/multi-value/blob/master/proposals/multi-value/Overview.md),
+  the [`try` blocks](#Try and catch blocks) may use values already in the stack.
```

Comment:
- Let's use 'multi-value' for consistency, because it's the official name of that proposal.
- What does the question mark after `br_on_exn` mean?
- The links do not work. They should be `#try-and-catch-blocks` and `#exception-data-extraction` respectively. You can check if they work by 'view file' menu on the upper right.
- In 'exceptions which consume one value', it's not very clear to me what 'consume' means. How about 'exceptions that contains one value' or something?
- ' Moreover, by using multi-value, the try blocks may use values already in the stack': 'try' also can push multiple values onto the stack. How about mentioning both?